Sources/Operators/ShareReplay.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ShareReplay.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 13/04/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher {
+    /// An overload on [Publisher.share](https://developer.apple.com/documentation/combine/publisher/3204754-share)
```

Comment:
Maybe instead of overload we should just say it's a variation of the share operator? 
I'd also remove the links from the documentation TBH since Xcode doesn't actually knows how to parse these and they create a bit of noise when trying to read the notes (same for other pieces of comments). WDYT?

Reply:
Seems to parse fine, at least client-side. I’m a fan of linking out, but, if they’re too noisy in your opinion, I’m game to remove them.

<img width="525" alt="Screen Shot 2020-04-18 at 8 21 31 PM" src="https://user-images.githubusercontent.com/1276296/79674558-3fa28a00-81b2-11ea-8c3c-f73fda5fa0ed.png">

Reply:
Ah cool! Probably some of the new markdown features :) 
I'd say - leave links that are references to other important code-things like `share()`, but I'd remove links to educational non-code content (like FIFO on Wikipedia) .

Also in this specific case might be worth saying 'A variation on `share()`, since it's a documentation on the operator iself (the method) and not the publisher  (concrete type)